adapters/seedingAlliance/seedingAlliancetest/supplemental/status_bad_request.json

```diff
@@ -0,0 +1,72 @@
+{
+    "mockBidRequest": {
+        "id": "test-request-id",
+        "imp": [
+            {
+                "id": "test-imp-id",
+                "banner": {
+                    "format": [
+                        {
+                            "w": 300,
+                            "h": 250
+                        },
+                        {
+                            "w": 300,
+                            "h": 600
+                        }
```

Comment:
Why would this be considered as a bad request?

Reply:
That request itself is trivial/meaningless - I'm just mocking a _Bad Request_ response to test the _MakeBids_ method to accomplish a higher code test coverage.